pallets/gear-debug/src/mock.rs

```diff
@@ -150,6 +151,7 @@ impl pallet_gear::Config for Test {
     type GasPrice = GasConverter;
     type WeightInfo = ();
     type OutgoingLimit = OutgoingLimit;
+    type CostMultiplier = ConstMultiplier;
```

Comment:
is it cost or const

Reply:
It is `cost` for the time being, so this is a typo. The actual/proper name for this syscall/functionality is not known yet. Any ideas?

Reply:
Reopen in once it is known then